commands/system/env.sh

```diff
@@ -9,15 +9,17 @@ function system:env() {
     echo OSTYPE=${OSTYPE}
     echo SOURCE=${SOURCE}
     echo ROOT_DIR=${ROOT_DIR}
-    echo DATA_DIR=${DATA_DIR}
-    echo CERT_DIR=${CERT_DIR}
+    _logYellow "DATA_DIR (Deprecated, use DDE_DATA_HOME) = ${DATA_DIR}"
```

Comment:
@sbaerlocher since its not a user-setting, we can fully remove data_dir from the code and just copy the ./data directory as boilerplate to the new path. This would also allow us to ship some default configuration